src/solCompile.ts

```diff
@@ -82,6 +83,7 @@ function formatInput(fileName: string): SolcInput {
 function cliCompile(
   input: SolcInput,
   solcVersion: string,
+  options?: CompilationOptions,
```

Comment:
Why is options optional here?

Reply:
For compatibility, we call that in *compileSolFileAndExtractContracts* function. I kind of didn't like the idea of modifying the call to pass some empty object there, so in cliCompile if nothing is passed then it just add no option to the solc exec command